sefaz/nsu/errors.go

```diff
@@ -0,0 +1,11 @@
+package nsu
+
+import "github.com/arquivei/foundationkit/errors"
+
+var (
+	// ErrCodeNotFound should be returned by repository when the NSU was not found for the given stakeholder
+	ErrCodeNotFound = errors.Code("NSU_NOT_FOUND")
```

Comment:
I believe that this error should not be here, as this has a deeper tie with domain logic.